test/unit/server.js

```diff
@@ -211,6 +211,65 @@ describe('Server', function () {
     });
   });
 
+  describe('_unsubscribe', function () {
+    var server;
+    var queues = ['a'];
+
+    beforeEach(function () {
+      // runnable-hermes 6.1.0 introduced .getQueues
+      sinon.stub(hermes, 'hermesSingletonFactory').returns({
+        unsubscribe: noop,
+        getQueues: sinon.stub().returns(queues)
+      });
+      server = new ponos.Server({ queues: queues });
+      server.setAllTasks({ a: noop });
+      sinon.stub(server.hermes, 'unsubscribe');
```

Comment:
replace `unsubscribe: noop` w/ `unsubscribe: sinon.stub()` above. then I don't think you need the `server.hermes.unsubscribe.restore()` below, either...